JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,19 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  if (seconds > 0) {
+    let secondsT = seconds % 60;
```

Comment:
in the name of variable `secondsT` the `T` could mean different things for other devs, so I'd recommend to use more descriptive variables name